resources/views/listing/partials/filter/selected/swatch.blade.php

```diff
@@ -1,5 +1,5 @@
 {{-- TODO: Display swatches correctly here --}}
 
-<template v-else-if="item.filter.input === 'swatch'">
-    @{{ refinement.label }}
+<template v-else-if="item.filter.super">
+    @{{ refinement.swatch.label }}
```

Comment:
Currently it is possible for swatch.label to be null due to https://github.com/rapidez/core/pull/718/files#diff-862a032d6992ba9a24c4278dd9a2ff425bfadcef76f9f721673d86967b5175f8R245

What is the best way to solve this? Use `refinement.label` as a fallback?